apps/core/scenes/bookingDetail/menuGroups/Passengers/Bag.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from 'react';
+import { Text, Icon, StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { View } from 'react-native';
+
+import type { Bag as BagType } from './__generated__/Bag.graphql';
+
+type Props = {|
+  +data: ?BagType,
+|};
+
+const getBagType = (quantity: number, type: string): string => {
+  let bagType = '';
+  const CABIN_BAG = 'Cabin bag';
+  const PERSONAL_ITEM = 'Personal item';
+  const CHECKED_BAGGAGE = 'Checked baggage';
+  switch (type) {
+    case 'CABIN_BAG':
+      bagType = CABIN_BAG;
+      break;
+    case 'PERSONAL_ITEM':
+      bagType = PERSONAL_ITEM;
+      break;
+    case 'CHECKED_BAGGAGE':
+      bagType = CHECKED_BAGGAGE;
+      break;
+    default:
+      break;
+  }
+  const plural = quantity > 1 && bagType !== CHECKED_BAGGAGE ? 's' : '';
+  return `${quantity} ${bagType}${plural}`;
+};
+
+const Bag = (props: Props) => {
+  const quantity = props.data?.quantity || 0;
+  const type = props.data?.type || '';
```

Comment:
Could you rather use the nullish coalescing operator instead (`??`) instead of `||`

Reply:
Changed now